src/docs/markdown/caddyfile/options.md

```diff
@@ -811,13 +806,29 @@ Also included is the [`proxy_protocol`](/docs/json/apps/http/servers/listener_wr
 			proxy_protocol {
 				timeout 2s
 				allow 192.168.86.1/24 192.168.86.1/24
+				deny 10.0.0.0/8
+				fallback_policy reject
 			}
 			tls
 		}
 	}
 }
 ```
 
+- **timeout** specifies the maximum duration to wait for the PROXY header. Defaults to `5s`.
+
+- **allow** is a list of CIDR ranges of trusted sources to receive PROXY headers. Unix sockets are trusted by default and not part of this option.
+
+- **deny** is a list of CIDR ranges of trusted sources to reject PROXY headers from.
+
+- **fallback_policy** is the action to take if the PROXY header comes from an address that not in either list of allow/deny. The default fallback policy is `IGNORE`. Accepted values of `fallback_policy` are:
```

Comment:
Could mention "case insensitive"

Reply:
I was afraid of encouraging the users, but fine if we can commit to it.

Reply:
Well the example had it as lowercase `fallback_policy reject` :joy: if you prefer just making that consistent instead of mentioning insensitive, that's fine too

Reply:
I think if we document it as lowercase it might be more Caddyfile-like, we don't usually use uppercase for constants/options in Caddyfile.

Reply:
Done